### PR TITLE
allow cors for localhost (#62)

### LINKED ISSUE #62: Ogarniecie CORS do komunkiacji z poza localhost:8000

### PR DESCRIPTION
**Pull request by ostatni5**
fix building and formating server

Closes #62

### CODE REVIEW

File: server/server.ts [hunk 1]
```diff
@@ -1,10 +1,16 @@
 import { Application, send } from "https://deno.land/x/oak/mod.ts";
 import * as path from "https://deno.land/std/path/mod.ts";
+import { oakCors } from "https://deno.land/x/cors/mod.ts";
 import router from "./routes.ts";
 
 const PORT = 8000;
 const app = new Application();
 
+app.use(
+    oakCors({
```
Comment (Petros9) [inline server/server.ts:10]: co robi funkcja oakCors?

File: src/lecturer/services/backEnd/BackEndService.tsx [hunk 2]
```diff
@@ -29,10 +29,7 @@ const createLecture = () => {
     return fetch(`${BASE_URL}/lectures`, {
         method: "POST",
         //TODO fix cors
-        mode: 'cors',
-        headers: {
```
Comment (Petros9) [inline src/lecturer/services/backEnd/BackEndService.tsx:33]: czy headers było od początku niepotrzebne, czy teraz się takie stało?

File: server/server.ts (hunk 1 above)
Comment (ostatni5) [reply server/server.ts:10]: It is a function from [CORS module](https://github.com/tajpouria/cors) for Deno, that simplifies CROS setup on the server.

File: src/lecturer/services/backEnd/BackEndService.tsx (hunk 2 above)
Comment (ostatni5) [reply src/lecturer/services/backEnd/BackEndService.tsx:33]: They were unnecessary.
I thought that CORS is a client-side problem and headers should be set here.

**Review by rozchlastywacz — APPROVED** [review]
Great job, although @michal367 might be unhappy with that code cleanup where you put every function export in new line, but I am not and as i said - great job mate!

**Review by Petros9 — APPROVED** [review]

**Pull request merged**